include/regs.h

```diff
@@ -118,6 +118,8 @@
 #define R_MESSAGE_DEBUGGER_TEXTID                YREG(79)
 #define R_C_UP_ICON_X                            YREG(88)
 #define R_C_UP_ICON_Y                            YREG(89)
+#define R_ENTER_RIDING_HORSE                     AREG(6)
```

Comment:
I had to read the code to see what this name really meant. I think something like SPAWN_ON_HORSE would have made it more clear to me. This one can be named after epona specifically too, right?

Reply:
if it is true that this is epona only, I would prefer `R_SPAWN_ON_EPONA` or `R_SPAWN_RIDING_EPONA`

Reply:
I don't know about this. I'm hesitant to say "spawn on horse/Epona" because of that extra conditional check on having obtained Epona, and it's also not like you set this flag to appear on a horse/epona during cutscenes. 

Testing it again, a more accurate explanation of what AREG(6) does is it tracks that the player rode a horse (either saddle or Epona) into an loading zone exit, as this is the only time it's ever set to 1. So even though there is only the one check that will always spawn Epona, I don't feel it necessarily has to be associated with Epona.

Reply:
Ok, I guess I can compromise with "horse" instead of "epona", I dont care too much.

But I do still think it needs a different name. I was quite confused by what "Enter on Epona" meant and had to read the code myself to understand it.
Spawn makes it immediately clear to me, and I dont think the edge case of cutscenes matters. But if you have another name idea that would fix my problem then sure Id like to hear.